lib/parser/current.rb

```diff
@@ -65,10 +65,19 @@ def warn_syntax_deviation(feature, version)
     require 'parser/ruby25'
     CurrentRuby = Ruby25
 
+  when /^2\.6\./
+    current_version = '2.6.0'
+    if RUBY_VERSION != current_version
+      warn_syntax_deviation 'parser/ruby26', current_version
+    end
+
+    require 'parser/ruby26'
+    CurrentRuby = Ruby26
+
   else # :nocov:
     # Keep this in sync with released Ruby.
-    warn_syntax_deviation 'parser/ruby25', '2.5.x'
-    require 'parser/ruby25'
-    CurrentRuby = Ruby25
+    warn_syntax_deviation 'parser/ruby26', '2.6.x'
```

Comment:
... and revert this change, this PR can be merged right now, and other work landed on top of it. This is what we generally do.